src/software/thunderscope/robot_diagnostics/chicker.py

```diff
@@ -73,6 +95,86 @@ def __init__(self, proto_unix_io):
         self.geneva_value = 3
         self.power_value = 1
 
+    def kick_clicked(self):
+        # if button is enabled
+        if self.kick_button_enable:
+            # send kick primitive
+            self.send_command(ChickerCommandMode.KICK)
+            self.toggle_kick_button()
+
+            # set and start timer to re-enable kick button after 3 seconds
+            self.start_timer_once(self.toggle_kick_button, 3 * MILLISECONDS_PER_SECOND)
```

Comment:
From my understanding we want to disable kick/chip buttons if one of the auto buttons is selected. Though there's a scenario where if kick/chip is selected, then within 3 seconds one of the auto buttons is selected, the kick/chip is enabled again due to the timer. One solution could be to check whether one of the auto radio buttons is selected before toggling kick/chip in `toggle_kick_button`. There could be other solutions as well.
![chicker_widget_buttons](https://user-images.githubusercontent.com/28585597/201190257-9c87dfe7-cf8b-40c0-b48d-8d1ab72fff9a.gif)